pom.xml

```diff
@@ -10,10 +10,11 @@
 	<url>https://github.com/Azure/azure-service-bus-java</url>
 	
 	<properties>
-		<proton-j-version>0.22.0</proton-j-version>
+		<proton-j-version>0.29.0</proton-j-version>
 	  	<junit-version>4.12</junit-version>
 		<slf4j-version>1.7.0</slf4j-version>
 		<client-current-version>2.0.0-PREVIEW-5</client-current-version>
```

Comment:
Changing this version to preview-6

Reply:
@binzywu Are you changing it or should i make another commit?